pennylane/ops/qubit/qchem_ops.py

```diff
@@ -90,10 +80,8 @@ def circuit(phi):
     grad_method = "A"
     """Gradient computation method."""
 
-    grad_recipe = four_term_grad_recipe
-    """Gradient recipe for the parameter-shift method."""
-
     parameter_frequencies = [(0.5, 1.0)]
+    """Frequencies with which parameter enter circuit."""
```

Comment:
This doesn't sound quite right to me 🤔 

Having said that, I probably wouldn't bother too much re: adding docstrings here, since they are not documented in the other files

Reply:
Is it just the article and singular-s missing or is it an invalid formulation? :thinking: 
"Frequencies with with the parameter enters a circuit."

Reply:
> "Frequencies with which the parameter enters a circuit."

This sounds better! Although maybe

> "Frequencies of each operation parameter with respect to an expectation value"

maybe is more accurate? Since 'circuit' is not defined here (although neither is expectation value).